index.html

```diff
@@ -43,30 +43,33 @@
 
     <div class="top-div">
       <div class="container pb-3">
-        <ul class="list-unstyled">
-          <li>
-            <i class="fas fa-map-marker-alt text-secondary"></i>
-            <a href="https://yandex.ru/maps/org/dve_pyatyorki/1024420856/?ll=20.509270%2C54.727694&oid=1024420856&ol=biz&z=18">
+        <div class="row">
+          <ul class="list-unstyled col-sm-6">
+            <li>
+              <i class="fas fa-map-marker-alt text-secondary"></i>
+              <a href="https://yandex.ru/maps/org/dve_pyatyorki/1024420856/?ll=20.509270%2C54.727694&oid=1024420856&ol=biz&z=18">
+                <span>
+                  Калининград, улица Горького, 55
+                </span>
+              </a>
+            </li>
+            <li>
               <span>
-                Калининград, улица Горького, 55
+                <i class="far fa-clock text-secondary" ></i>
               </span>
-            </a>
-          </li>
-          <li>
-            <span>
-              <i class="far fa-clock text-secondary" ></i>
-            </span>
-            с 09:00 до 19:00
-            <span id="text-open" class="text-success d-none">
-               (Сейчас открыто)
-            </span>
-          </li>
-        </ul>
+              с 09:00 до 19:00
+              <span id="text-open" class="text-success d-none">
+                 (Сейчас открыто)
+              </span>
+            </li>
+          </ul>
+          <img class="col-sm-6 img-fluid" src="./assets/ readme/background.jpg" alt="Торговый центр «Две пятерки»" title="Торговый центр «Две пятерки»">
```

Comment:
what the reason to use `img-fluid` class?

Reply:
in order to make image size responsive, to scale image size